tasks/init.rb

```diff
@@ -0,0 +1,62 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
nit: I don't think this should be here

Reply:
Does it automatically work out the interpreter on UNIX? I found the (somewhat undocumented) logic that does it on windows, I would have thought it wouldn't do that on UNIX because it's not necessary.

Reply:
Not 100% sure.  I guess we can leave it in.  I assumed you tried this on Mac and *nix derivative

Reply:
Have not tested on mac, will though